ribir/Cargo.toml

```diff
@@ -58,11 +58,11 @@ raw-window-handle.workspace = true
 [package.metadata.release]
 tag = true
 pre-release-replacements = [
-  {file="../CHANGELOG.md", search="@Unreleased", replace="{{version}}"},
-  {file="../CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", min=0},
-  {file="../CHANGELOG.md", search="@ReleaseDate", replace="{{date}}"},
-  {file="../CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [@Unreleased] - @ReleaseDate", exactly=1},
-  {file="../CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[@Unreleased]: https://github.com/RibirX/Ribir/compare/{{tag_name}}...HEAD", exactly=1},
-  {file="../docs/zh/起步/创建一个应用.md", search="@RIBIR_VERSION", replace="{{version}}"},
-  {file="../docs/en/get_started/creating_an_application.md", search="@RIBIR_VERSION", replace="{{version}}"},
+  {file="../CHANGELOG.md", search="@Unreleased", replace="{{version}}", prerelease = true},
+  {file="../CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", min=0, prerelease = true},
+  {file="../CHANGELOG.md", search="@ReleaseDate", replace="{{date}}", prerelease = true},
+  {file="../CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [@Unreleased] - @ReleaseDate", exactly=1, prerelease = true},
+  {file="../CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[@Unreleased]: https://github.com/RibirX/Ribir/compare/{{tag_name}}...HEAD", exactly=1, prerelease = true},
+  {file="../docs/zh/起步/创建一个应用.md", search="@RIBIR_VERSION", replace="{{version}}", prerelease = true},
```

Comment:
The file path seems to have changed?

Reply:
No errors found, can you point me to the file location you think is correct?